ui/packages/components/src/Timeline/TimelineNode/TimelineNode.tsx

```diff
@@ -146,9 +146,10 @@ function useOutput({
       return;
     }
 
-    // We should only fetch output if the node has ended. There are some edge
-    // cases where a node can have an output item ID but not be in an end state
-    if (!isEndStatus(nodeStatus)) {
+    // We should only fetch output if the node has ended or errored ("errored"
+    // means there will be a retry). There are some edge cases where a node can
+    // have an output item ID but not be in an end state
+    if (!isEndStatus(nodeStatus) && nodeStatus != 'errored') {
```

Comment:
Would be good to use constants here instead of string matching, but this UI will be replaced soon enough, so it's ok.

Reply:
It's still type safe since `nodeStatus` is a literal union